torch/csrc/cuda/python_comm.cpp

```diff
@@ -0,0 +1,45 @@
+#include "torch/csrc/utils/pybind.h"
+#include "torch/csrc/cuda/comm.h"
+
+#include <chrono>
+
+static const char *_broadcast_docstr =
```

Comment:
Put doc strings in Python (in `_torch_docs.py`) or similar.

Reply:
I couldn't because pybind puts its own docstring on the function, and our utility to set them complains about this